catalyst/callbacks/quantization.py

```diff
@@ -1,3 +1,54 @@
+from typing import Dict, Optional, TYPE_CHECKING, Union
+from pathlib import Path
+
+import torch
+
+from catalyst.core import Callback, CallbackNode, CallbackOrder
+from catalyst.utils import BestModel, quantize_model
+
+if TYPE_CHECKING:
+    from catalyst.core import IRunner
+
+
+class QuantizationCallback(Callback):
+    def __init__(
+        self,
+        metric: str,
+        minimize_metric: bool = True,
+        qconfig_spec: Dict = None,
+        dtype: Union[str, Optional[torch.dtype]] = "qint8",
+        logdir: Union[str, Path] = None,
+        filename: str = "quantized.pth",
+    ):
+        """
+        @TODO
+        Args:
+            metric:
+            minimize_metric:
+            qconfig_spec:
+            dtype:
+            logdir:
+            filename:
+        """
+        super().__init__(order=CallbackOrder.External, node=CallbackNode.master)
```

Comment:
btw, for model postprocessing, we have `CallbackOrder.ExternalExtra` order (not the best name, I know :) )